### PR TITLE
coding style improvements

### PR DESCRIPTION
**Pull request by wurstchristoph**
- short array syntax
- double emtpy lines
- opening curly braces
- unused use statements
- space indents

### CODE REVIEW

**Comment by wurstchristoph** [comment]
we should also get rid of ```_someMethod()``` naming conventions like in https://github.com/owncloud/mail/blob/master/lib/cache/cache.php#L97

what do you think @DeepDiver1975? seems like you introduced those ;-)

File: lib/controller/messagescontroller.php [hunk 1]
```diff
@@ -159,7 +159,7 @@ public function getHtmlBody($messageId) {
 
 			return new HtmlResponse($html);
 		} catch(\Exception $ex) {
-			return new TemplateResponse($this->appName, 'error', array('message' => $ex->getMessage()), 'none');
+			return new TemplateResponse($this->appName,'error', ['message' => $ex->getMessage()], 'none');
```
Comment (DeepDiver1975) [inline lib/controller/messagescontroller.php]: keep the space please

File: lib/service/html.php [hunk 2]
```diff
@@ -96,7 +95,8 @@ public function __construct() {
 	 * @return string
 	 */
 	public function convertLinks($data) {
-		$regex = "/(ht|f)tp(s?)\:\/\/(([a-zA-Z0-9\-\._]+(\.[a-zA-Z0-9\-\._]+)+)|localhost)(\/?)([a-zA-Z0-9\-\.\?\,\'\/\\\+&amp;%\$#!_]*)?([\d\w\.\/\%\+\-\=\&amp;\?\:\\\&quot;\'\,\|\~\;]*)/";
```
Comment (DeepDiver1975) [inline lib/service/html.php]: I'd prefer to keep this string in one line - on concat of regex a char is easily lost and your will not recognize - THX

**Comment by DeepDiver1975** [comment]
> we should also get rid of _someMethod() naming conventions like in https://github.com/owncloud/mail/blob/master/lib/cache/cache.php#L97
> 
> what do you think @DeepDiver1975? seems like you introduced those ;-)

that class is a pure copy of a Horde class - I'd prefer to keep it unchanged

**Comment by DeepDiver1975** [comment]
https://github.com/horde/horde/blob/master/framework/Imap_Client/lib/Horde/Imap/Client/Cache/Backend.php

**Comment by wurstchristoph** [comment]
@DeepDiver1975 thanks for your feedback!
Reverted cache.php modifications and fixed the other two things :-)

**Comment by jancborchardt** [comment]
@DeepDiver1975 @zinks- @irgendwie @clementhk  can you review this?

File: lib/controller/folderscontroller.php [hunk 3]
```diff
@@ -137,7 +137,7 @@ public function create() {
 
 			$newFolderId = $mailbox;
 			return new JSONResponse(
-				array('data' => array('id' => $newFolderId)),
+				array('data' => ['id' => $newFolderId]),
```
Comment (zinks-) [inline lib/controller/folderscontroller.php]: a stray array

Comment (wurstchristoph) [inline lib/controller/folderscontroller.php]: @zinks- thanks, removed it ;-)

**Comment by zinks-** [comment]
Good calls, although I prefer the array() syntax but that's a matter of taste :+1:

**Comment by DeepDiver1975** [comment]
:+1:

**Pull request merged**